babs/utils.py

```diff
@@ -1939,6 +1995,43 @@ def check_job_account(job_id_str, job_name, username_lowercase):
     jobs under qw, r, etc, or does not exist (not submitted);
     Also, the current username should be the same one as that used for job submission.
     """
+    if type_system == "sge":
+        return _check_job_account_sge(job_id_str, job_name, username_lowercase)
+    elif type_system == "slurm":
+        return _check_job_account_slurm(job_id_str, job_name, username_lowercase)
+
+
+def _check_job_account_slurm(job_id_str, job_name, username_lowercase):
+    """
+    get information for a finished job in Slurm by calling `sacct`
+    """
+    proc_qacct = subprocess.run(
+        ["sacct", "-u", username_lowercase,
+         "-j", job_id_str],
+        stdout=subprocess.PIPE
+    )
+
+    proc_qacct.check_returncode()
+    msg_l = proc_qacct.stdout.decode('utf-8').split("\n")
+    msg_head = msg_l[0].split()
+    if "State" not in msg_head or "JobID" not in msg_head or "JobName" not in msg_head:
+        return "sacct doesn't provide information about the job"
```

Comment:
Even if `sacct` is empty for a job, printed message will still include the head of the table - it looks like this on my end: I'm using a non-existing job ID from my account:
```
% sacct -j 12345 -u $USER
JobID           JobName  Partition    Account  AllocCPUS      State ExitCode 
------------ ---------- ---------- ---------- ---------- ---------- --------
```
So I'm adding additional checks like if there is a real job row in PR #96.